src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -30,6 +32,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   }
 
   @Override
+  public void onStart() {
```

Comment:
OK. 

Just for my info, why is it too late in render?

And if it is too late can it be done in OnCreateDialog then? Just trying to figure out why it makes sense to put this in OnStart?